### PR TITLE
Prepare requirements for tflite

### PR DESCRIPTION
**Pull request by mtrofin**
We need to get past TF 2.9, but 2.10 isn't released yet, and there are
compat issues between 2.10-RC1 and tfagents. The recommended approach is
to install nightlies of these.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -20,7 +20,7 @@ h5py==3.6.0
 idna==3.2
 keras==2.7.0
```
Comment (ebrevdo) [inline requirements.txt:21]: i think you can remove these?  keras*

Comment (ebrevdo) [reply requirements.txt]: that's my guess.  let's retrigger tests and see?

Comment (mtrofin) [reply requirements.txt]: huh. now they fail with the `StructuredValue` failure. Not _quite_ the same as https://github.com/google/ml-compiler-opt/runs/7932290113?check_suite_focus=true, though.

File: compiler_opt/rl/constant_value_network.py [hunk 2]
```diff
@@ -45,9 +45,8 @@ def __init__(self, input_tensor_spec, constant_output_val=0, name=None):
 
     self._constant_output_val = constant_output_val
 
-  def call(self, observation, step_type=None, network_state=(), training=False):
+  def call(self, inputs, step_type=None, network_state=(), training=False):
```
Comment (mtrofin) [inline compiler_opt/rl/constant_value_network.py:48]: @yundiqian this takes care of 

```
compiler_opt/rl/constant_value_network.py:48:2: W0237: Parameter 'inputs' has been renamed to 'observation' in overridden 'ConstantValueNetwork.call' method (arguments-renamed)
```

Anything I'm missing?

**Comment by ebrevdo** [comment]
LGTM

**Review by ebrevdo — APPROVED** [review]

File: compiler_opt/rl/constant_value_network.py (hunk 2 above)
Comment (yundiqian) [reply compiler_opt/rl/constant_value_network.py:48]: the change looks good to me, original warning due to some trivial tf agents hassles

**Review by yundiqian — APPROVED** [review]

**Pull request merged**